### PR TITLE
Added version unit tests

### PR DESCRIPTION
**Pull request by TheHollidayInn**
Simply adding some unit tests to the version command. I just want to make sure I am on the right track here before adding more. Thanks!

### CODE REVIEW

File: kubos/test/test_version.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        kubos_version = ''
+        get_active_kubos_version.return_value = kubos_version
+
+        version.execCommand(None, None)
+
+        get_repo.assert_called()
+        get_tag_list.assert_called()
+        print_tag_list.assert_called()
+        calls = [
+            call('Kubos-CLI version    : %s' % 'v' + '0.2.2'),
```
Comment (kyleparrott) [inline kubos/test/test_version.py]: I'd like to keep from having to hard code the cli version in this test. We've already bumped the version on [kubostech/kubos-cli](https://github.com/kubostech/kubos-cli/blob/master/module.json#L3) to 0.2.3.

File: kubos/test/test_version.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    @patch('kubos.utils.git_utils.get_active_kubos_version')
+    @patch('logging.info')
+    def test_exec_command_with_no_version_and_no_directory(self, info, get_active_kubos_version):
+        kubos_version = ''
+        get_active_kubos_version.return_value = kubos_version
+
+        version.execCommand(None, None)
+
+        calls = [
+            call('Kubos-CLI version    : %s' % 'v' + '0.2.2'),
```
Comment (kyleparrott) [inline kubos/test/test_version.py]: Same comment as above.

**Comment by kyleparrott** [comment]
I've also fiddled with our CircleCI settings so it should start running tests automatically for forks of the kubostech/kubos-cli repo.

File: kubos/test/test_version.py (hunk 1 above)
Comment (TheHollidayInn) [inline kubos/test/test_version.py]: Make sense to me!

**Pull request merged**